setup.cfg

```diff
@@ -0,0 +1,6 @@
+[isort]
```

Comment:
Can we have a `setup.cfg` in https://github.com/collective/buildout.plonetest/tree/c54dbac141e4bbef9f4d84580b1e8a7bc1a6881b as well to be a canonical version (and a comment in the beginning of the file stating that it was from there)?

Reply:
I don't think that's possible.